seqr/views/apis/data_manager_api.py

```diff
@@ -424,31 +430,61 @@ def validate_callset(request):
 
 @pm_or_data_manager_required
 def get_loaded_projects(request, sample_type, dataset_type):
-    projects = get_internal_projects().filter(
-        family__individual__sample__sample_type=sample_type, is_demo=False,
-    ).distinct().order_by('name').values('name', projectGuid=F('guid'), dataTypeLastLoaded=Max(
+    projects = get_internal_projects().filter(is_demo=False)
+    project_samples = None
+    if dataset_type == Sample.DATASET_TYPE_VARIANT_CALLS:
+        project_samples = _fetch_airtable_loadable_project_samples(request.user)
+        projects = projects.filter(guid__in=project_samples.keys())
+        exclude_sample_type = Sample.SAMPLE_TYPE_WES if sample_type == Sample.SAMPLE_TYPE_WGS else Sample.SAMPLE_TYPE_WGS
+        projects = projects.exclude(family__individual__sample__sample_type=exclude_sample_type)
+    else:
+        projects = projects.filter(family__individual__sample__sample_type=sample_type)
```

Comment:
```suggestion
    if dataset_type == Sample.DATASET_TYPE_VARIANT_CALLS:
        project_samples = _fetch_airtable_loadable_project_samples(request.user)
        projects = projects.filter(guid__in=project_samples.keys())
       
   projects = projects.filter(family__individual__sample__sample_type=sample_type)
```
This jumped out to me as a possible simplification but I bet using `exclude` here was intentional - why? SNV_INDEL projects should also return RNA samples?

Reply:
so the difference is that explicitly filtering for the sample type only includes projects that already have data loaded for that sample type. Excluding the opposite sample type includes projects that are loaded for the sample type AND projects that have no loaded data at all. For SV and MITO we only want to be able to add to projects that have existing data so they use the `filter` logic, but for SNV_INDEL data at some point we do need to load it for the first time, and so we need to instead use this `exclude` logic

Reply:
thanks for the explanation. can you add this as a comment?